kinesis-video-gst-demo/kinesis_video_gstreamer_audio_video_sample_app.cpp

```diff
@@ -158,6 +160,11 @@ typedef struct _CustomData {
 
     uint32_t total_track_count;
 
+    bool use_absolute_fragment_times;
+
+    // Pts of first frame
+    uint64_t first_pts;
```

Comment:
should be per track in case of audio video. may happen that audio video use different clocks.

Reply:
We were relying on the audio video getting sync clock from gstreamer before, if setting both  to start from time diff 0 then the audio video frame can have timestamp overlapping and failure in producer sdk.

Reply:
ok